crates/ghost_actor/src/ghost_actor.rs

```diff
@@ -73,12 +73,12 @@ impl<
     }
 
     /// see GhostContextEndpoint::request
-    pub fn request(
+    pub fn request<A: Any>(
```

Comment:
Do we want to change this to something like `UserData: Any`?

Reply:
Done in https://github.com/holochain/lib3h/pull/273/commits/ee7046d0a9c9a447067b4a24c4b14691089de164